nixos/modules/programs/xppen.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib; let
+  cfg = config.programs.xppen;
+in
+
+{
+  options.programs.xppen = {
+    enable = mkEnableOption (lib.mdDoc "XPPen PenTablet application");
+    package = mkPackageOption pkgs "xppen" { };
+  };
+
+  config = mkIf cfg.enable {
+    hardware.uinput.enable = true;
+
+    environment.systemPackages = [
+      cfg.package
+    ];
+
+    services.udev.packages = [ cfg.package ];
+
+    systemd.services.xppen-create-config-dir = {
```

Comment:
This service seems like something that might be better achieved with systemd-tmpfiles

Reply:
tmpfiles are not necessarily temporary, despite the name.